### DISCUSSION TITLE
Low RealtimeCap on OnePlus 8

### DISCUSSION THREAD

**Discussion by rudgeman** [NightscoutFoundation/xDrip#2311] [Q&A]
For reference I am already using the recommended settings and have tried messing with different Samsung settings on and off. I have battery optimization off for xdrip and Bluetooth. Have even tried turning them off using Activity Launcher. Using the nightly build.

My RealtimeCap has been at roughly 65% on a One Plus 8 since updating to Android 12. Was working very well (>95%) before the update, which is a shame since 12 finally fixed BT tethering. Would love any other suggestions or settings recommendations to improve this number. I have read through other threads about inconsistent connectivity and signal loss but so far have not found a resolution.

Thinking of even switching phones to something that has less collection inconsistency.

**Comment by Navid200** [comment]
Would you please post a screenshot of the system status page?  You can blackout the transmitter ID if you like.

**Reply by rudgeman** [reply]
![IMG_20220906_123408](https://user-images.githubusercontent.com/35583446/188688941-ead4c9fc-336d-4e37-8755-dc26754a5569.jpg)
![Screenshot_2022-09-06-12-33-44-50_2dd2c1bcb8f659cd696bcba2072f751b](https://user-images.githubusercontent.com/35583446/188688944-d7161385-377d-46cd-841e-47eb7ebfc3c2.jpg)

I also added the history page showing the low cap number.

**Reply by Navid200** [reply]
Would you please post a screenshot of the following page?
xDrip Settings &#8722;> G5/G6 Debug Settings

**Reply by rudgeman** [reply]
![Screenshot_2022-09-06-13-02-25-90](https://user-images.githubusercontent.com/35583446/188696703-a5a05f78-aa81-43e5-ab02-2056ba0a20c8.jpg)
Should match the recommended settings for Android 10+

**Comment by jamorham** [comment]
The slow wakeup is the problem, the phone framework doesn't comply with the android specification. See if any of these suggestions help:
https://dontkillmyapp.com/oneplus

**Reply by rudgeman** [reply]
Will give them a try and report back, thank you for the suggestion and even more so the amazing app!

**Reply by rudgeman** [reply]
Unfortunately no luck yet. I had done most of these except for disabling app auto launch. Will continue searching for OP specific solutions and post here if I find one that works.
